res/css/views/rooms/_TopUnreadMessagesBar.pcss

```diff
@@ -22,60 +22,73 @@ limitations under the License.
-    border-radius: 16px;
-    background-color: $secondary-accent-color;
-    border: 6px solid $accent;
-    pointer-events: none;
-}
+    &::after {
+        content: "";
+        position: absolute;
+        top: -8px;
+        left: 10.5px;
+        width: 4px;
+        height: 4px;
+        border-radius: 16px;
+        background-color: $secondary-accent-color;
+        border: 6px solid $accent;
+        pointer-events: none;
+    }
 
-.mx_TopUnreadMessagesBar_scrollUp {
-    height: 38px;
-    border-radius: 19px;
-    box-sizing: border-box;
-    background: $background;
-    border: 1.3px solid $muted-fg-color;
-    cursor: pointer;
-}
+    /* Define common declarations. Their values are specified below. */
+    .mx_TopUnreadMessagesBar_scrollUp,
+    .mx_TopUnreadMessagesBar_markAsRead {
+        background: var(--background);
+        width: var(--width);
+        height: var(--height);
+        border: var(--border);
+        border-radius: var(--borderRadius);
 
-.mx_TopUnreadMessagesBar_scrollUp::before {
-    content: "";
-    position: absolute;
-    width: 36px;
-    height: 36px;
-    mask-image: url("$(res)/img/element-icons/message/chevron-up.svg");
-    mask-repeat: no-repeat;
-    mask-size: 20px;
-    mask-position: center;
-    background: $muted-fg-color;
-}
+        --background: $background;
+        --border: 1.3px solid $muted-fg-color;
 
-.mx_TopUnreadMessagesBar_markAsRead {
-    display: block;
-    width: 18px;
-    height: 18px;
-    background: $background;
-    border: 1.3px solid $muted-fg-color;
-    border-radius: 10px;
-    margin: 5px auto;
-}
+        &::before {
+            content: "";
+            position: absolute;
+            background: var(--background);
+            width: var(--width);
+            height: var(--height);
+            mask-repeat: no-repeat;
+            mask-position: center;
+            mask-image: var(--maskImage);
+            mask-size: var(--maskSize);
 
-.mx_TopUnreadMessagesBar_markAsRead::before {
-    content: "";
-    position: absolute;
-    width: 18px;
-    height: 18px;
-    mask-image: url("$(res)/img/cancel.svg");
-    mask-repeat: no-repeat;
-    mask-size: 10px;
-    mask-position: 4px 4px;
-    background: $muted-fg-color;
+            --background: $muted-fg-color;
+        }
+    }
+
+    .mx_TopUnreadMessagesBar_scrollUp {
+        --height: 38px;
+        --borderRadius: 19px;
+
+        box-sizing: border-box;
+        cursor: pointer;
+
+        &::before {
+            --width: 36px;
+            --height: 36px;
+            --maskImage: url("$(res)/img/element-icons/message/chevron-up.svg");
+            --maskSize: 20px;
+        }
+    }
+
+    .mx_TopUnreadMessagesBar_markAsRead {
+        --width: 18px;
+        --height: 18px;
+        --borderRadius: 10px;
+
+        display: block;
+        margin: 5px auto;
+
+        &::before {
+            --maskImage: url("$(res)/img/cancel.svg");
+            --maskSize: 10px;
+        }
```

Comment:
`--width: 18px` and `--height: 18px` do not need to be repeated here, as they are inherited from `mx_TopUnreadMessagesBar_markAsRead`.